syncmaster/backend/settings/__init__.py

```diff
@@ -41,6 +35,15 @@ class BackendSettings(SyncmasterSettings):
         SYNCMASTER__SERVER__DEBUG=True
     """
 
+    TZ: str = "UTC"
+    crypto_key: str = Field(description="Key for encrypt/decrypt credentials data")
```

Comment:
Could we move this field to a separated class like CredentialsEntryptionSettings? `SYNCMASTER__CRYPTO_KEY` name does not look very descriptive.